src/Selenium2Driver.php

```diff
@@ -656,12 +656,18 @@ public function setValue($xpath, $value)
         $value = strval($value);
 
         if (in_array($elementName, array('input', 'textarea'))) {
-            $existingValueLength = strlen($element->attribute('value'));
+            $deleteSequence = Key::CONTROL . 'a' . KEY::NULL_KEY . Key::COMMAND . 'a' . KEY::NULL_KEY;
```

Comment:
Sending both shortcuts doesn't sound quite right. Maybe we'd better stay with OS detection as you suggested, but use remote capabilities for retrieval of user agent like here: http://mashtips.com/get-system-os-browser-version-java-selenium-webdriver/